test/providers/store/mongoDefinitionPagination.js

```diff
@@ -32,8 +32,11 @@ const shouldPaginateSearchCorrectly = function () {
     })
 
     after('cleanup database', async function () {
-      await mongoStore.collection.drop()
-      await mongoStore.close()
+      if (mongoStore) {
+        await mongoStore.collection.drop()
```

Comment:
I found during some local testing that the `before()` function didn't always complete before `after()` which would mean the variable `mongoStore` was still `undefined` and this line would throw.